packages/legacy-workbench/src/css/aclmanagement.css

```diff
@@ -220,23 +220,23 @@ input::placeholder {
 }
 
 .acl-tag-allow {
-    background-color: var(--color-success-light);
-    color: var(--color-success-dark);
+    background-color: var(--gw-success-lighter);
+    color: var(--gw-success-base);
```

Comment:
Hm... isn't there text color that corresponds to --gw-success-lighter background?